crt_portal/cts_forms/templates/landing.html

```diff
@@ -98,7 +100,12 @@ <h1 class="h1__display">{% trans "We uphold the civil rights of <em>all</em> peo
           <p class="crt-landing--largetext">
             <strong>{% trans "If you believe your civil rights, or someone else’s, have been violated, submit a report using our online form." %}</strong>
           </p>
+          {% if shutdown_mode == True %}
```

Comment:
nit: direct comparison to bool

```suggestion
          {% if shutdown_mode %}
```